benchmark/timers/timers-cancel-pooled.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+var common = require('../common.js');
+var assert = require('assert');
+
+var bench = common.createBenchmark(main, {
+  thousands: [500],
+});
+
+function main(conf) {
+  var N = +conf.thousands * 1e3;
```

Comment:
exceedingly minor nit... why `N`? I tend to dislike caps for variables that aren't `const`